### PR TITLE
Migrate to .NET 5

### PR DESCRIPTION
**Pull request by bjss-chris**
+semver:major

Activity: [AB#11089](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11089)
Task: [AB#11090](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11090)

### CODE REVIEW

File: tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs [hunk 1]
```diff
@@ -21,8 +21,9 @@ public SolutionEpicEntityBuilder()
 
         public enum SolutionEpicStatus
         {
+            Undefined = 0,
             Passed = 1,
-            NotEvidenced = 3
+            NotEvidenced = 3,
```
Comment (PhysicsJon) [inline tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs:26]: Did we have a status of 2 at some point?

**Review by PhysicsJon — APPROVED** [review]

File: tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs (hunk 1 above)
Comment (bjss-chris) [reply tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs:26]: I think we do but I can only assume it's not used in the tests. I think there are a few things like this in BAPI – one for some further tech debit investigation at some point I think.

Comment (bjss-chris) [reply tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs:26]: Actually, the correct values are:

```
Passed = 1,
NotEvidenced = 2,
Failed = 3,
```

so I'm not sure why the values are as they are. I think I might add another tech debt task to look into these enum definitions in the test projects.

Comment (bjss-chris) [reply tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs:26]: Added tech debt task, 11129.

**Pull request merged**